cmd/zfs/zfs_main.c

```diff
@@ -4240,9 +4240,21 @@ zfs_do_receive(int argc, char **argv)
 			}
 			break;
 		case 'd':
+			if (flags.isprefix) {
+				(void) fprintf(stderr, gettext("invalid option "
+				    "combination: -d and -e are mutually "
+				    "exclusive\n"));
```

Comment:
this message is a little misleading if you, say, provide `-e` multiple times.

Reply:
This should be fixed with the latest push, thank you for catching it!